lib/actions/ssh.coffee

```diff
@@ -98,11 +78,20 @@ module.exports =
 			.then ({ username, uuid, containerId, proxyUrl }) ->
 				throw new Error('Did not find running application container') if not containerId?
 				Promise.try ->
+					sshProxyCommand = ''
+					proxyConfig = global.PROXY_CONFIG
+					if proxyConfig
+						{ proxyAuth } = proxyConfig
+						proxyHost = "-p #{proxyConfig.host}:#{proxyConfig.port}"
+						proxyAuth = if proxyAuth then "-P #{proxyAuth}" else ''
```

Comment:
Good catch, I've used the -P from some article on the net, could it be renamed in a newer version of proxytunnel?

Speaking about escaping - good point as well, needs investigation